### PR TITLE
Core notes of 2022-{10-14,11-04,12-02}

### PR DESCRIPTION
**Pull request by lschuermann**
### Pull Request Overview

Some post-deadline core team call notes, in the 3-for-1 super saver pack! For the notes of December 2nd I think I accidentally transcribed the wrong call, but this one didn't have anyone volunteering for notes, so needed someone to transcribe it anyways. There should be another December notes PR following this one shortly.

### Testing Strategy

This pull request was tested by spellcheck and skimming the notes.

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [ ] Ran `make prepush`.

### CODE REVIEW

**Review by hudson-ayers — APPROVED** [review]
Thanks for doing all of these!

File: doc/wg/core/notes/core-notes-2022-11-04.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+  "Copyright <year> Tock Developers"), that seems fine. What about copyright
+  assignments to individuals?
+
+* Leon: I presume this is fine. In many countries, it is actually not possible
+  to "transfer" copyright. As long as we have each file and each contribution
+  consistently licensed (under MIT and Apache-2.0), commiters can retain their
+  copyright but give a non-revocable, non-exclusive usage agreement as defined
+  by these licenses.
+
+* Johnathan: Alex, would you be okay with a generic copyright assignment such as
```
Comment (jrvanwhy) [inline doc/wg/core/notes/core-notes-2022-11-04.md:177]: `copyright assignment` -> `copyright statement`

Comment (lschuermann) [reply doc/wg/core/notes/core-notes-2022-11-04.md]: Thanks!

**Review by hudson-ayers — APPROVED** [review]

**Pull request merged**